src/modules/identity/RadixSimpleIdentity.spec.ts

```diff
@@ -0,0 +1,43 @@
+import { expect } from 'chai'
+import 'mocha'
+import { RadixIdentityManager, RadixTransactionBuilder } from '../..'
+import { RadixAtom } from '../atommodel'
+
+describe('RadixSimpleIdentity', () => {
+
+    it('Different identities leave different signatures on atom', async () => {
+        const atom1 = new RadixAtom()
+        const atom2 = new RadixAtom()
+        const manager = new RadixIdentityManager()
+        const identity1 = manager.generateSimpleIdentity()
+        const identity2 = manager.generateSimpleIdentity()
+
+        await identity1.signAtom(atom1)
+        await identity2.signAtom(atom2)
+
+        const signature1 = atom1.signatures[identity1.address.getUID().toString()]
+        const signature2 = atom2.signatures[identity2.address.getUID().toString()]
+
+        expect(signature2.r.toString()).is.not.equal(signature1.r.toString())
+        expect(signature2.s.toString()).is.not.equal(signature1.s.toString())
+    })
+
+    it('Identities from same seed leave same signature on atom', async () => {
+        const seed = 'abc'
+        const atom1 = new RadixAtom()
+        const atom2 = new RadixAtom()
+        const manager = new RadixIdentityManager()
+        const identity1 = manager.generateSimpleIdentityFromSeed(Buffer.from(seed))
+        const identity2 = manager.generateSimpleIdentityFromSeed(Buffer.from(seed))
+
+        await identity1.signAtom(atom1)
+        await identity2.signAtom(atom2)
+
+        const signature1 = atom1.signatures[identity1.address.getUID().toString()]
+        const signature2 = atom2.signatures[identity2.address.getUID().toString()]
+
+        expect(signature2.r.toString()).is.equal(signature1.r.toString())
+        expect(signature2.s.toString()).is.equal(signature1.s.toString())
```

Comment:
This feels like a naive approach to me. Surely there must be a better way to verify signature. Any comments @MuncleUscles

Reply:
You should be able to do `expect(signature1).to.deep.equal(signature2)`, same elsewhere

But yes, a more valuable test would be to check if the signature verifies properly. 
RadixAddress has a .verify method, if you pass it `atom.getHash()` and the signature it should succeed

Reply:
Sweet! The "deep equals" did the trick!

The `verify` was also valuable, but there seems to be some issues with the current implementation. I have proposed a suggested solution in another PR. Let's continue this discussion there.